babel.config.js

```diff
@@ -115,50 +130,17 @@ module.exports = {
         ['transform-react-remove-prop-types', { mode: 'remove' }],
       ],
     },
-    es: {
-      plugins: [
-        'transform-react-constant-elements',
-        'transform-dev-warning',
-        ['react-remove-properties', { properties: ['data-mui-test'] }],
-        [
-          'transform-react-remove-prop-types',
-          {
-            mode: 'wrap',
-          },
-        ],
-      ],
-      // It's most likely a babel bug.
-      // We are using this ignore option in the CLI command but that has no effect.
-      ignore: ['**/*.test.js'],
```

Comment:
This was used in the cli with `--ignore **/*.test.js`. Most shells already expand those. If you want to pass a glob to a node script you have to escape it with `"`. I've made that mistake myself and it's quite common. Only [prettier documents this ](https://prettier.io/docs/en/cli.html) well.